recipe-server/client/control/components/recipes/PreferenceExperimentFields.js

```diff
@@ -49,7 +49,16 @@ export default class PreferenceExperimentFields extends React.PureComponent {
           >
             <DocumentUrlInput disabled={disabled} />
           </FormItem>
+
+          <FormItem
+            label="High volume recipe?"
+            name="arguments.isHighVolume"
+            initialValue={recipeArguments.get('isHighVolume')}
+          >
+            <Checkbox />
```

Comment:
This ends up looking pretty bad. I'm not sure how to make it look better. @andymikulski do you have any idea for how to improve this?

![screenshot-2017-10-24 shield control panel](https://user-images.githubusercontent.com/305049/31970640-86ee309a-b8cd-11e7-8f7d-b84aa3f6efa1.png)

Reply:
I think up until this point we've been using our custom [SwitchBox](https://github.com/mozilla/normandy/blob/master/recipe-server/client/control/components/forms/SwitchBox.js) component (which uses an Ant [Select](https://ant.design/components/switch/) under the hood). I think we'll end up having to do something similar for Checkboxes.

I don't imagine it'd be a huge thing to implement, most of the SwitchBox code would already apply. Alternatively, if you're interested in landing this sooner than later, you could simply swap the `Checkbox` here with a `SwitchBox` and we can change the components later once the checkbox stuff is written.